BookPlayer/Player/PlayerManager.swift

```diff
@@ -10,6 +10,8 @@ import Foundation
 import AVFoundation
 import MediaPlayer
 
+// swiftlint:disable file_length
```

Comment:
travis is failing due to swiftlint, `PlayerManager` isn't over 400 lines, so we can delete this comment safely

Reply:
Artifact from extracting this PR from my other work, should be fixed now.